nbpages/converter.py

```diff
@@ -281,6 +308,13 @@ def process_notebooks(nbfile_or_path, exec_only=False, exclude=[], include=[],
         if not exec_only:
             converted.append(nbc.convert())
 
+    # Create a test suite and write report to file
+    # Note: depending on system, mode may need to be set to wb
+    ts = [TestSuite("Convert", test_cases, package="spacetelescope_notebooks")]
+    with open('./convert_test_report.xml', mode='w') as file:
```

Comment:
Two things about this section of the code:
1) Maybe it makes sense to put this writing in the `NBPagesConverter` itself instead of this function? In my mind the function should be as simple as possible so that the converter can be used fairly flexibly programatically.
2) The file name should be an option.  If we go with 1 above, that could live as an attribute on `NBPagesConverter`, otherwise it has to get passed in somewhere to this function.

And one explicit suggestion:
```suggestion
    with open('convert_test_report.xml', mode='w') as file:
```
because then we don't have to worry about ``os.sep`` and platform cross-compatibility.

Reply:
How would this best be placed in NBPagesConverter? Looks like a new converter is created for each notebook (if its a path) and this segment of code has to run after all notebooks are executed to create a test suite that wraps the test cases (each notebook adds a test case).